### PR TITLE
My assignment1

### PR DESCRIPTION
**Pull request by LeonAppDev**
# Handler changed in hapi version 17.5.2
Since the right answer still use old version 16's method to create server and reply, need a new version to refelect changes on hapi 17.5.2

### CODE REVIEW

File: lib/index.js [hunk 1]
```diff
@@ -6,23 +6,22 @@ const Hapi = require("hapi");
 const Package = require("../package.json");
 //Make declaration
 
-(function (){
+(async ()=>{
```
Comment (zoe-1) [inline lib/index.js]: hapi style guide does not use [IIFE](https://developer.mozilla.org/en-US/docs/Glossary/IIFE) function calls to start the server.
See: [style guide regarding module globals](https://github.com/hapijs/contrib/blob/master/Style.md#module-globals)

File: lib/index.js [hunk 2]
```diff
@@ -6,23 +6,22 @@ const Hapi = require("hapi");
...
 
-       const server = new Hapi.server();
+       const server = new Hapi.server({port:Number(process.env.PORT||"8000")});
```
Comment (zoe-1) [inline lib/index.js]: See hapi style rules [for spaces](https://github.com/hapijs/contrib/blob/master/Style.md#spaces)) regarding spaces in objects `{}`.

**Comment by zoe-1** [comment]
See [sample solution](https://github.com/zoe-1/university-dev/blob/v0.1.1/lib/index.js) for more details.
In respect to how the hapi server is built and started, we need to build the server as a function which can be exported. Therefore, using the [IIFE](https://developer.mozilla.org/en-US/docs/Glossary/IIFE) function call does not work.  Future assignments will export the 'init' or 'start' function to a start script and test scripts.  Plus, if we plan to build multiple hapi services which communicate with each other, it is important to build the hapi servers / services in a way that another application can start and stop them (ie. manage the services). Exporting the init() allows for this.  This assignment does not require an init() to be exported. Just trying to explain some of the reasons for hapi's style preferences and why we don't use IIFE functions to start the server. Above [Module Global Styles](https://github.com/hapijs/contrib/blob/master/Style.md#module-globals) shows preferred style to init() the server.

**Comment by LeonAppDev** [comment]
Hi, Zoe
    Thanks for your good explanation, I have another project to run so it takes me so long to code according to the convention. 
     I add ESLint and apply convention rules using it.

**Pull request closed without merging**